zjit/src/cruby.rs

```diff
@@ -262,7 +262,7 @@ pub struct ID(pub ::std::os::raw::c_ulong);
 pub type IseqPtr = *const rb_iseq_t;
 
 // Given an ISEQ pointer, convert PC to insn_idx
-pub fn iseq_pc_to_insn_idx(iseq: IseqPtr, pc: *mut VALUE) -> Option<u16> {
+pub fn iseq_pc_to_insn_idx(iseq: IseqPtr, pc: *mut VALUE) -> Option<u32> {
```

Comment:
We're using u32 elsewhere for insn_idx, so I changed this to match. YJIT has https://github.com/ruby/ruby/blob/af6b98f7a25670cb569f6da59904b3c05482b16e/yjit/src/yjit.rs#L149, so if we want to do something similar I think we could use u16 everywhere instead.